contrib/mesos/pkg/executor/executor.go

```diff
@@ -47,7 +47,6 @@ import (
 
 const (
 	containerPollTime = 1 * time.Second
-	launchGracePeriod = 5 * time.Minute
 	podRelistPeriod   = 5 * time.Minute
```

Comment:
We still have the podStore in the executor, well hidden in the framework informer. So the constant still exists.